packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/DynamicZone/ComponentCategory.tsx

```diff
@@ -59,15 +60,23 @@ const ComponentCategory = ({
             </ComponentBox>
           ))}
         </Grid>
-      </Accordion.Content>
+      </ResponsiveAccordionContent>
     </Accordion.Item>
   );
 };
 
+const ResponsiveAccordionContent = styled(Accordion.Content)`
+  container-type: inline-size;
```

Comment:
Looks like we need to edit our eslint setup because ts-styled-plugin warns about this being an unknown rule. That package is deprecated so we need to replace it with @styled/typescript-styled-plugin

Reply:
I've gotten rid of the warning but I'm a bit confused. I couldn't find any dependency for `typescript-styled-plugin` (which is deprecated). I tried installing https://github.com/styled-components/typescript-styled-plugin, but it doesn't seem necessary to install it for it to work...

Adding the following config to `packages/core/content-manager/admin/tsconfig.json` does the trick but I don't understand how it's working if the plugin ins't installed 🤔 

```
    "plugins": [
      {
        "name": "@styled/typescript-styled-plugin",
        "lint": {
          "validProperties": ["container-type"],
          "unknownAtRules": "ignore"
        }
      }
    ]
```

Reply:
Huh that is a bit of a mystery indeed. Maybe the warning just comes from something with our vscode configs? Anyway I wouldn't recommend writing a reference to a package we know is not there 🤔

Reply:
Man it took me forever but I finally figured out that `@styled/typescript-styled-plugin` is installed through the vscode styled components extension, that's why this config is working. I also learned the vscode extension is no longer maintained :( https://github.com/styled-components/vscode-styled-components/issues/275. 

What do you think should I leave the config and add a comment explaining where the dep comes from? Or should I remove it and we live with the warning 🤔

Reply:
And just for the record...AI was no help...not that I'm keeping score or anything

Reply:
I think living with the warning is safer, I'm afraid referring that rule may break for people using other IDEs

Reply:
Sounds good, updated!